src/experimental-layers/src/bezier-curve-layer/bezier-curve-layer.js

```diff
@@ -25,7 +25,7 @@ import {GL, Model, Geometry} from 'luma.gl';
 import vs from './bezier-curve-layer-vertex.glsl';
 import fs from './bezier-curve-layer-fragment.glsl';
 
-const NUM_SEGMENTS = 40;
+const NUM_SEGMENTS = 80;
```

Comment:
I was thinking whether it should be part of the layer api that users can control the number of segments on this curve.